src/Worker/Delivery.php

```diff
@@ -564,7 +564,8 @@ private static function deliverMail($cmd, $contact, $owner, $target_item, $thr_p
 				$headers  = 'From: ' . Email::encodeHeader($local_user['username'],'UTF-8') . ' <' . $local_user['email'] . '>' . "\n";
 			}
 		} else {
-			$headers  = 'From: '. Email::encodeHeader($local_user['username'], 'UTF-8') . ' <noreply@' . DI::baseUrl()->getHostname() . '>' . "\n";
+			$sender = DI::config()->get('config', 'sender_email', 'noreply@' . DI::baseUrl()->getHostname());
```

Comment:
Here the local variable `$sender` is being used only once. Is that really wanted?

Reply:
I don't like used once variables as well, but the alternative is to break the below string concatenation on multiple lines which isn't ideal either.

Reply:
Same here, that is why maybe it needs fixing?